src/app/frontend/replicasetlist/replicasetlist.scss

```diff
@@ -47,3 +47,29 @@
 .kd-replicaset-card-label {
   margin-right: 1em;
 }
+
+.kd-menu-logs-md-menu-content {
+  min-width: 350px;
```

Comment:
Can you use built in width mechanism: https://material.angularjs.org/latest/demo/menu ? Instead of this 350px value